src/development/tools/devtools/memory.md

```diff
@@ -17,41 +17,48 @@ memory at a given moment.
-Resident Set Size (RSS). As you interact with your application various
-events are detected e.g., memory GC (garabage collection),Flutter events,
-user fired events (using the ```dart:developer``` package) are collected
-in the same timeline as the memory statistics. All of this collected
-statistics and events are displayed in charts see [Memory anatomy](#memory-anatomy).
+At the top-level, when the memory tab is selected,
+memory statistics from the VM are collected.
+These statistics are displayed in the two overview charts
+(Dart memory and Android-only) the collection of general memory usage
+e.g., total heap used, external heap, maximum heap capacity,
+and Resident Set Size (RSS).
+As you interact with your application various events are detected
+e.g., memory GC (garbage collection), Flutter events,
+user fired events (using the `dart:developer` package)
+are collected in the same timeline as the memory statistics.
+All of this collected statistics and events are displayed in charts.
+See [Memory anatomy](#memory-anatomy).
 
 ## Analysis and snapshots
 
-A Snapshot is a complete, the most complex and time consuming view of
-all objects in the Dart memory heap. Each time a snapshot is taken, an
-analysis is performed over the collected memory data.  The analysis
-attempts to identify any memory patterns that may cause leaks or lead to
-application crashes. For example, loading large assets for thumbnail-sized
-images inefficiently, memory usage can be improved by loading smaller
-assets or adjusting the cacheWidth/cacheHeight to decode an image to a
-smaller size reducing the memory usage of the ImageCache. The analysis
-catches issues like this see [Analysis tab](#analysis-tab).
+A Snapshot is a complete, the most complex and time-consuming view
+of all objects in the Dart memory heap. Each time a snapshot is taken,
+an analysis is performed over the collected memory data.
+The analysis attempts to identify any memory patterns
+that may cause leaks or lead to application crashes.
+For example, loading large assets for thumbnail-sized images inefficiently,
+memory usage can be improved by loading smaller assets
+or adjusting the `cacheWidth`/`cacheHeight` to decode an image
+to a smaller size reducing the memory usage of the ImageCache.
+The analysis catches issues like this.
+See [Analysis tab](#analysis-tab).
 
 ## Allocations and tracking
 
-Monitoring all allocations involes you directly interacting with DevTools
+Monitoring all allocations involves you directly interacting with DevTools
 and your application to isolate a short period of time that you are
 interested in knowing how many objects were allocated, how many bytes
 were allocated, or tracking all the places in your code where a particular
 class is allocated. This information is available under the "Allocations"
-tab of Memory profiler and his a fairly fast with less overhead than using
+tab of Memory profiler, and he's a fairly fast with less overhead than using
```

Comment:
@sfshaza2 i suspect your wordsmithing is required here

Reply:
Oh dear. I just scanned the whole page and, yes, this is in serious need of some wordsmithing. I plan to do that in another PR. Thanks, @AlexV525, for bringing this to my attention!!!

Reply:
This article exists for a long time, while I doing the improvement, I found many emotional sentences that can be improved too.